doc/api/readline.md

```diff
@@ -447,6 +447,10 @@ autocompletion is disabled when copy-pasted input is detected.
 
 If the `stream` is a [TTY][], then it must be in raw mode.
 
+*Note*: This is automatically called by any readline instance on its `input`
+if the `input` is a terminal. Closing the `readline` instance does not stop
```

Comment:
Not sure why I I'm being pinged on this (as opposed to anybody else) to be honest...